examples/media/schema.json

```diff
@@ -47,61 +46,64 @@
           "post": {
             "description": "media can be any media object. All `encapsulating` media objects like album, playlist etc. will be flattend to root level of mediacollection. E.g. if a playlist contains track A and B, the tracks A and B will be added to the mediacollection. Changing an existing mediaCollectionObject would mean a modification of its `items` property content (e.g. adding or removing items from an existing collection)",
             "parameters": {
-              "resource": "media.collections",
-              "method": "post",
-              "required": [
-              "items"
-              ],
               "properties": {
                 "items": {
-                  "description": "media identifier",
+                  "isRequired": true,
                   "type": "array",
                   "items": {
                     "type": "string",
                     "format": "uri"
                   }
                 }
               }
-            }
+            },
+            "scopesNeeded": [
+              "MEDIA_WRITE"
```

Comment:
1st: Should generally allow / be allowed to define own scopes. 
2nd: Capital letters is not a good idea (at least in my opinion).

Reply:
1st: scopes can of course be freely defined.
=> maybe we want to define the scopes in the Schema itself on root level? A service can define it's own scopes this way and it is sef contained.. If scopes are duplicated I  other services, this is fine.. No issue..

2nd: I am also fine with lowerCamelCase, just thought the imperative style of UPPER_CASE is widely adopted for auth scopes

Reply:
Didn't know 2nd. 
I always prefer definitions being self-contained ;-)

Reply:
I'm not a fan of mixing casing styles. Almost everywhere we use lowerCamelCase except in `scopesNeeded`.

Why isn't it called `accessControl`? `scopesNeeded` doesn't describe the purpose of this property.

Reply:
I just fixed the scope casing.. 

What about the naming now? do we have an agreement?